Cargo.toml

```diff
@@ -42,3 +40,7 @@ overflow-checks = true
 
 [profile.release.package.transfer-contract]
 overflow-checks = true
+
+[patch.crates-io]
+dusk-wallet-core = { path = "../wallet-core" }
```

Comment:
This local dependency and the entire patch.crates-io needs to go away

Reply:
For this I will need to release `wallet-core`, and as such I will need to release `rusk-abi`. I suggest an alpha for this. Here's [a PR](https://github.com/dusk-network/rusk/pull/826)